ansible/roles/postgresql/templates/create_postgres_dump.sh.j2

```diff
@@ -3,7 +3,7 @@ BACKUP_TYPE=$1
 DAYS_TO_RETAIN_BACKUPS=$2
 
 TODAY=$(date +"%Y_%m_%d")
-pg_dumpall --clean | gzip  > "{{ postgresql_backup_dir }}/postgres_$BACKUP_TYPE_$TODAY.db.gz"
+pg_dumpall --clean | gzip  > "{{ postgresql_backup_dir }}/postgres_${BACKUP_TYPE}_${TODAY}.db.gz"
```

Comment:
It don't think it could figure out where `$BACKUP_TYPE` ended so it was leaving it out. probably looking for a variable called `$BACKUP`. I did a bit of playing around and variables with underscores seem to only work correctly with the braces or at the end of a string